real_time/surface_src/java_src/Alice/src/test/java/TestSo2Pose.java

```diff
@@ -1,105 +1,109 @@
-import static org.junit.Assert.*;
-
-import org.junit.After;
-import org.junit.Before;
+import com.roboclub.robobuggy.map.So2Pose;
 import org.junit.Test;
 
-import com.roboclub.robobuggy.main.Util;
-import com.roboclub.robobuggy.map.So2Pose;
+import static org.junit.Assert.fail;
 
 
+/**
+ * Tests aspects of the So2Pose
+ */
 public class TestSo2Pose {
 
-	@Before
-	public void setUp() throws Exception {
-	}
-
-	@After
-	public void tearDown() throws Exception {
-	}
 
+	/**
+	 * basic test of the so2pose
+	 */
 	@Test
 	public void test() {
-		So2Pose A = new So2Pose(0.0, 0.0, 0.0);
-		So2Pose B = new So2Pose(0.0, 1.0, 0.0);
-		So2Pose C = A.mult(B);
-		if(C.getY() != 1.0){		
+		So2Pose a = new So2Pose(0.0, 0.0, 0.0);
+		So2Pose b = new So2Pose(0.0, 1.0, 0.0);
+		So2Pose c = a.mult(b);
+		if(c.getY() != 1.0){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2369456130/issues/source?bid=2938665&fileBranchId=3216826#l21)